tools/scylla-sstable.cc

```diff
@@ -1464,15 +1472,17 @@ class scylla_metadata_visitor : public boost::static_visitor<> {
             _writer.Key("exclusive");
             _writer.Bool(e.left.exclusive);
             _writer.Key("token");
-            _writer.String(disk_string_to_string(e.left.token));
+            auto left_token = disk_string_to_token(e.left.token);
+            _writer.String(left_token.to_sstring());
```

Comment:
i am just curious, can we format the token as an int64? like
```c++
_writer.Int64(left_token.raw());
```

the difference is 

1. json formats a string like `"30"`
2. and it formats a number like `30`
3. and `to_sstring()` considers the maximum token as `numeric_limits<int64_t>::min()`, while `raw()` take the maximum token as `numeric_limits<int64_t>::max()`.

Reply:
token::minimum() and token::maximum() are used for denoting positions before and after all keys, respectively. It doesn't matter much in the context of sstable, because first and last tokens will be a result of hashed keys.

Reply:
i see. so, in other words, we can just go with `_writer.Int64(left_token.raw());` ? or we have good reasons to format the token as hex string?

Reply:
Yes, my intuition was to use to_sstring() which is supposed to be provide the external world with textual representation of the token.